### PR TITLE
Fix flaky engine api acceptance test

### PR DESCRIPTION
**Pull request by siladu**
Assert on objects but use prettyString for failure message

### CODE REVIEW

**Review by jframe — APPROVED** [review]
LGTM

File: acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/AbstractJsonRpcTest.java [hunk 1]
```diff
@@ -95,7 +95,10 @@ public void test() throws IOException {
     final ObjectNode actualBody = JsonUtil.objectNodeFromString(response.body().string());
     final ObjectNode expectedBody =
         JsonUtil.objectNodeFromString(testCase.getResponse().toString());
-    assertThat(actualBody.toPrettyString()).isEqualTo(expectedBody.toPrettyString());
+    assertThat(actualBody)
+        .withFailMessage(
+            "%s\ndid not equal\n %s", actualBody.toPrettyString(), expectedBody.toPrettyString())
+        .isEqualTo(expectedBody);
```
Comment (garyschulte) [inline acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/AbstractJsonRpcTest.java:101]: it was an ordering thing from prettyPrint?

**Review by garyschulte — APPROVED** [review]
LGTM

File: acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/AbstractJsonRpcTest.java (hunk 1 above)
Comment (siladu) [reply acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/AbstractJsonRpcTest.java:101]: yeh, not sure why

**Pull request merged**